core/Symbols.cc

```diff
@@ -2201,14 +2201,16 @@ uint32_t ClassOrModule::hash(const GlobalState &gs) const {
     if (!members().empty()) {
         // Rather than use membersStableOrderSlow, which is... slow..., use an order dictated by symbol ref ID.
         // It's faster to sort, and SymbolRef IDs are stable during hashing.
+        // TODO(jez) Why is this necessary, if everything already hashes its owner?
```

Comment:
What does hashing the owner have to do with producing a stable hash (which is what you get by sorting the members in some order)?

Reply:
i think my question to myself was "why does a class's hash depend on any of it's members?"

in any case, i think we'll revisit this as we move more and more symbol kinds to take the fast path unconditionally.